addons/mail/static/src/models/chat_window/chat_window.js

```diff
@@ -185,7 +189,7 @@ function factory(dependencies) {
              * @returns {integer}
              */
             const _getNextIndex = index => {
-                const directionOffset = reverse ? -1 : 1;
+                const directionOffset = reverse ? 1 : -1;
```

Comment:
Isn't this only change enough?

Reply:
This fix the issue on LTR but not in RTL.